test/e2e/authorImage.spec.ts

```diff
@@ -0,0 +1,26 @@
+import test, { expect } from "@playwright/test";
+import { BASE_URL,VALID_EMAIL,VALID_PASSWORD} from "../helper-config";
+import { LoginPage } from "../pages/LoginPage";
+import { AuthorImage } from "../pages/AuthorImagePage";
+
+let authorImage: AuthorImage;
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(BASE_URL);
+  const loginPage = new LoginPage({ page });
+  await loginPage.navigate();
+  await loginPage.login(VALID_EMAIL, VALID_PASSWORD);
+  authorImage = new AuthorImage({ page });
+});
+
+test("should add author image", async ({ page }) => {
+  await authorImage.add();
+  await authorImage.finishedButtonForAdd.waitFor();
```

Comment:
that should also happen in the page-object, just put it inside the `add` function